### PR TITLE
At 417 support multiple configurations

### PR DESCRIPTION
**Pull request by munyofrey**
Adds the ability to have a secondary configuration so that data is sent to two stitch accounts at once.

### CODE REVIEW

File: lib/mini_stitch.rb [hunk 1]
```diff
@@ -31,11 +31,17 @@ def self.status
...
     def initialize
       @token = nil
       @client_id = nil
+      @secondary_token = nil
+      @secondary_client_id = nil
+    end
+
+    def has_second_configuration? 
+      !secondary_token.nil? && !secondary_client_id.nil?
     end
```
Comment (danishmughal) [inline lib/mini_stitch.rb:45]:
```suggestion
    def has_second_configuration? 
      secondary_token.present? && secondary_client_id.present?
    end
```

File: lib/mini_stitch/api.rb [hunk 2]
```diff
@@ -6,11 +6,11 @@ class Api
     def initialize(table_name, sequence, key_names, data)
       validate(key_names, data, table_name)
 
-      @request_params = { Authorization: "Bearer #{MiniStitch.configuration.token}" }.merge(DEFAULT_REQUEST_PARAMS)
       @table_name = table_name
       @sequence = sequence.to_i
       @key_names = key_names.map(&:to_s)
-      @data = build_records(data)
+      @data = build_records(data, MiniStitch.configuration.client_id)
+      @secondary_data = build_records(data, MiniStitch.configuration.secondary_client_id)
```
Comment (danishmughal) [inline lib/mini_stitch/api.rb:13]: Maybe this step can be skipped `if !MiniStitch.configuration.has_second_configuration?` ?

**Review by danishmughal — APPROVED** [review]
LGTM!

File: lib/mini_stitch.rb (hunk 1 above)
Comment (munyofrey) [reply lib/mini_stitch.rb:45]: Unfortunately `.present?` is a rails method and can not be used here.

Comment (danishmughal) [reply lib/mini_stitch.rb:45]: TIL. I've been working in Rails land for too long.

**Pull request merged**